src/main/kotlin/com/vk/admstorm/AdmStormStartupActivity.kt

```diff
@@ -38,6 +41,15 @@ class AdmStormStartupActivity : ProjectActivity {
             return
         }
 
+        //TODO: remove it after release AdmStorm on Windows
+        if (!SystemInfo.isWindows) {
+            AdmErrorNotification("AdmStorm is not available on Windows yet").withActions(
```

Comment:
Where's the part about the disabling?

Reply:
I don't think this is a good idea, many apps require you to close notifications all the time and it doesn't annoy anyone. Then, after release, we will need to add additional code to clean this up, then further remove this code, and so on. I think this is too redundant

Reply:
No need to torment using unnecessary notifications. This spoils the user experience.
To delete the code, it's minutes of time.